keps/sig-network/1024-nodelocal-cache-dns/kep.yaml

```diff
@@ -1,4 +1,4 @@
-title: graduate-nodelocaldns-to-beta
+title: NodeLocal DNS Cache
 kep-number: 1024
 authors:
   - "@prameshj"
```

Comment:
add yours too?

Reply:
I don't want to take credits for others work. I didn't do anything creative here.